app/views/wikies/search_results.html.erb

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html>
+<head>
+<div class="container">
+ 
+<h3>Search_results for : </h3><h5>'<%= params[:search_keywords]%>'</h5>
+
+<% if @found_products.empty? %>
+<p>We are sorry, no results were found</p>
+<% end %>
+
+<table>
+  <thead>
+    <tr>
+      <th>Title</th>
+      <th>Body</th>
+      <th></th>
+      <th colspan="3"> </th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <% @found_products.each do |wiky| %>
+      <tr>
+        <td><%= wiky.title %></td>
+        <td><%= wiky.body %></td>
+        <td><%#= User.find(wiky.user_id).email %></td>
+       <td><%= link_to 'Show', wiky %></td>
+        <td><%= link_to 'Edit', edit_wiky_path(wiky) %></td>
+        <td><%= link_to 'Destroy', wiky, method: :delete, data: { confirm: 'Are you sure?' } %></td>
+    
+      </tr>
+    <% end %>
+  </tbody>
+</table>
+<br>
+</head>
+</html>
```

Comment:
Invalid HTML, this file shouldn't need the doctype, and the opening and closing tags for `html`, `head`. These tags are already specified in the application layout (`layouts/application.html.erb`) that will wrap this view template. Check the result by using "View Source" in the browser for the search results page.